### PR TITLE
feat(rpc/otterscan): set total_fees in ots_getBlockDetails

### PR DESCRIPTION
**Pull request by jsvisa**
wait for alloy new release ref https://github.com/alloy-rs/alloy/pull/1043

### CODE REVIEW

**Review by mattsse — APPROVED** [review]
ty!

File: crates/rpc/rpc/src/otterscan.rs
Comment (Rjected) [inline crates/rpc/rpc/src/otterscan.rs:55]: looks like the lint err is blocked on alloy bump, which should be coming soon

**Review by Rjected — APPROVED** [review]
lgtm, will need a rebase tomorrow after the bump

**Review by mattsse — APPROVED** [review]

**Pull request merged**
